dashboard/app/models/authentication_option.rb

```diff
@@ -171,6 +171,7 @@ def update_oauth_credential_tokens(credentials)
 
     other = User.find_by_email_or_hashed_email(email)
     if other && other != user
+      return if other.authentication_options.length == 1 && other.authentication_options.first&.lti?
```

Comment:
Would this be cleaner if it were:

```
return if other.authentication_options.all?(&:lti?)
```

Can an account ever have multiple LTI options?

Reply:
You're right. An account can have multiple LTI auth options. There is a helper function to check for this, but it will also need to be updated. Thanks for the review!